discover_overlay/locales/fr/LC_MESSAGES/default.po

```diff
@@ -109,13 +109,13 @@ msgid "Right"
 msgstr "Droite"
 
 msgid "Settings"
-msgstr "Paramètres"
+msgstr "Ouvrir les Paramètres"
 
 msgid "Close Overlay"
-msgstr "Fermer l’overlay"
+msgstr "Désactiver l’overlay"
```

Comment:
The source string says "close" but that translation means "deactivate", the original translation is correct IMO.

Reply:
To me 'close' feels more natural in English than 'quit' or 'deactivate', but I'm not sure if the same would be true in French. I'm more interested in languages conveying the meaning fluidly than a direct-from-EN translation.

Reply:
The difference is that close is just once and you could reopen while deactivate implies it will not open again, if you see what I mean. I think quit implies better that it just stops the process and you have to start it again, meanwhile close is usually just close the window.